src/ResourceManagement/api-specs.json

```diff
@@ -51,7 +51,7 @@
     "dir": "CosmosDB",
     "source": "specification/cosmos-db/resource-manager/readme.md",
     "package": "Microsoft.Azure.Management.CosmosDB.Fluent",
-    "args": "--payload-flattening-threshold=1 --tag=package-2019-08-preview"
+    "args": "--payload-flattening-threshold=1 --tag=package-2019-08-preview --ignore-request-uri-redirect"
```

Comment:
What is the reason to ignore?

Reply:
The old issue https://github.com/Azure/azure-libraries-for-net/blob/master/src/ResourceManagement/CosmosDB/dev-note.md

Now I just add the param into autorest.csharp forked repo.